### ISSUE TITLE
Toggling 'relevance' no more visible in entry list

### ISSUE THREAD

**Issue by ytzemih** [JabRef/jabref#5834]
Marking/unmarking entries has no more effect. Except for the changes in the bibtex source, nothing is currently shown. This also holds of quality, print status, priority, read status (as far as existing in 4.x).

Is that feature currently being revised, thus, things that I should see are just not implemented or is this a bug?

(I'm using Ubuntu 19.04 and JabRef 5.0-beta.342--2020-01-15--58709e7, Linux 5.3.0-26-generic amd64, Java 13.0.2)

**Comment by calixtus** [comment]
Hi @ytzemih . Thank you for your note.

Marking an entry is no more in v. 5.0.
Instead we use entry groups, which provide the same functionality. If you're changing from v. 4.x to 5.0, JabRef should put the marked entries in a new group.

Special fields like ranking should be visible in the entry editor as a simple field (will probably be changed in the future for a more easy-to-use alternative) and in the main table in a special column in al manner. You can add the colums to the main table via the options menu > preferences > entry table colums .

Can you please try this and tell us, if this solves this issue?

**Comment by calixtus** [comment]
Please do not forget to make a backup of your library before you try a beta.

**Comment by ytzemih** [comment]
Will do and thanks for the clarification. Yes, I tried that feature, in the entry editor and I added columns to the entry table. Seems to work all nicely.

Just a minor issue: The ranking stars can currently only be altered through the context menu, unlike read status, relevance, priority, and quality which can be directly toggled. I suppose an iteration by clicking would be sufficient if the widget doesn't allow to directly alter the stars based on the position of the mouse click.
